src/Lecture1.hs

```diff
@@ -67,7 +70,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int 
+lastDigit n = n `mod` 10
```

Comment:
That's correct! 💯 But, unfortunateley, you implementation doesn't work on negative numbers, because `mod` for negative numbers behaves differently 😥 

You can improve the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```

So the final result will look like this

```suggestion
lastDigit n = abs n `mod` 10
```